php/src/Data/ConfigurationManager.php

```diff
@@ -83,6 +83,30 @@ public function GetLastBackupTime() : string {
         return $lastBackupTime;
     }
 
+    public function GetBackupTimes() : array {
+        if (!file_exists(DataConst::GetBackupArchivesList())) {
+            return $array[] = '';
+        }
+        
+        $content = file_get_contents(DataConst::GetBackupArchivesList());
+        if ($content === "") {
+            return $array[] = '';
+        }
+
+        $backupLines = explode("\n", $content);
```

Comment:
```suggestion
        $backupLines = explode('\n', $content);
```

To be consistent with the rest of the code